qemu/tests/cfg/multi_disk.cfg

```diff
@@ -60,6 +60,9 @@
                 stg_params = "drive_format:virtio,scsi-hd,usb2"
             usbs += " default-ehci"
             usb_type_default-ehci = usb-ehci
+            Host_RHEL.m6:
+                usbs = "default-ehci"
+                usb_type_default-ehci = ich9-usb-ehci1
```

Comment:
1. uhci is added.
    -device ich9-usb-ehci1,id=default-ehci,addr=1d.7,multifunction=on,bus=pci.0 \
    -device ich9-usb-uhci1,id=default-ehci.0,multifunction=on,masterbus=default-ehci.0,addr=1d.0,firstport=0,bus=pci.0 \
    -device ich9-usb-uhci2,id=default-ehci.1,multifunction=on,masterbus=default-ehci.0,addr=1d.2,firstport=2,bus=pci.0 \
    -device ich9-usb-uhci3,id=default-ehci.2,multifunction=on,masterbus=default-ehci.0,addr=1d.4,firstport=4,bus=pci.0 \

2. I have thought that way, but such configurations will be updated when there are more new supports, like rhel.8